src/crawler/crawlers/arso_water_slovenija/crawler.js

```diff
@@ -1,159 +1,173 @@
 
 class ArsoWaterSloveniaCrawler {
+    constructor(water_type, url_file, data_file) {
+        this.water_type = water_type;
+        this.config = CrawlerUtils.loadConfig(__dirname);
 
-    constructor(water_type, URL_file, data_file) {
-        this.water_type = water_type
         if (this.water_type == 'sw') {
-            this.URL = 'http://www.arso.gov.si/vode/podatki/amp/Ht_30.html'
-            this.base_URL = 'http://www.arso.gov.si/vode/podatki/amp/'
-            this.time_parse = 'DD.MM.YYYY hh:mm'
+            this.url = this.config.surface_water.start;
+            this.base_url = this.config.surface_water.base;
+            this.time_parse = 'DD.MM.YYYY hh:mm';
         } else if (this.water_type == 'gw') {
-            this.URL = 'http://www.arso.gov.si/vode/podatki/podzem_vode_amp/podt_30.html'
-            this.base_URL = 'http://www.arso.gov.si/vode/podatki/podzem_vode_amp/'
-            this.time_parse = 'DD-MM-YYYY hh:mm'
+            this.url = this.config.ground_water.start;
+            this.base_url = this.config.ground_water.base;
+            this.time_parse = 'YYYY-MM-DD hh:mm';
         }
-        this.URL_file=URL_file
-        this.data_file=data_file
+        this.url_file = url_file;
+        this.data_file = data_file;
     }
 
-    // run to crawl
-    crawl() {
-        let links = null
-        let self = this
-        fs.readFile(self.data_file + self.URL_file, 'utf-8', function(err, dat) {
-            if (err) {
-            } else {
-                links = dat.trim().split('\n')
+    /**
+     * Run to crawl.
+     */
+    async crawl() {
+        let self = this;
+        let links =  await this.getURLs();
 
-                for (let i = 0; i < links.length - 1; i++) {
-                    let URL = links[i]
-                    self.getData(URL)
-                }
-            }
-        })
+        for (let i = 0; i < links.length - 1; i++) {
+            let url = links[i];
+            self.getData(url);
+        }
     }
 
-    // construct file with URLs with data
-    getURLs(URL=this.URL, base_URL=this.base_URL){
-        let self = this
-        fs.writeFile(self.data_file + self.URL_file, '', function (err) {})
-        request(URL, function (err, res, body) {
-            if(err)
-            {
-                console.log(err, "error occured while hitting URL");
-            }
-            else
-            {
-                const $ = cheerio.load(body)
-                $('map>area').each(function() {
-                    var link = $(this).attr('href');
-                    
-                    link = base_URL + link
-                    fs.appendFile(self.data_file + self.URL_file, link + '\n', function (err) {})
-                })
-            }
-        });
+    /**
+     * construct file with URLs with data
+     * @param {String} url 
+     * @param {String} base_url
+     */
+    getURLs(url = this.url, base_url = this.base_url) {
```

Comment:
Use the function in CrawlerUtils, there is no need to reimplement the same code. Do parsing outside of this function.